pkg/option/config.go

```diff
@@ -287,6 +287,9 @@ type daemonConfig struct {
 	// EnvoyLogPath specifies where to store the Envoy proxy logs when Envoy
 	// runs in the same container as Cilium.
 	EnvoyLogPath string
+
+	// EnableSockOps specifies whether to enable sockops (socket lookup).
+	EnableSockOps bool
```

Comment:
hmm must missing something don't we also need a populateConfig update as well to pull in the string from the command line and/or the defaults pkg value?

Reply:
Something like this I would guess,
```
option.Config.EnableSockOps = viper.GetBool(option.SockopsEnableName)
```
Would also be nice to either call it SockopsEnable or rename the others `SockopsEnableName->EnableSockOpsName` and `SockopsEnable->SockopsEnableDflt`. If you want I can write a patch to put in front of this to do the name translation. I don't really care if its SockopsEnable or EnableSockops as long as its consistent.

Reply:
You're right, good catch. Fix coming...

Reply:
I'll change it to `SockopsEnable` for consistency as well.

Reply:
Fixed in latest upload.